packages/tests/untagged-ids/src/entities/Author.test.ts

```diff
@@ -26,4 +26,17 @@ describe("Author", () => {
     await em.flush();
     expect(a1.idOrFail.startsWith("a:")).toBe(false);
   });
+
+  it("can load author and books", async () => {
+    const em = newEntityManager();
+    const a1 = newAuthor(em);
+    const a2 = newAuthor(em);
+    const b1 = newBook(em, { author: a1 });
+    const b2 = newBook(em, { author: a2 });
+    await em.flush();
+    await em.refresh();
```

Comment:
I'm going to go ahead and merge the PR, to get this released :rocket: , but fwiw the `em.refresh()`s are probably unnecessary here...

Typically `refresh` is only necessary if some other database connection/transaction/em has modified the data that `em` had already loaded, and it now wants to reload.

90% of our usage of `refresh` is in test cases to do things like:

* Start one `em1` to setup test data
* Create a 2nd "clean" `em2` and invoke the endpoint-under-test (this ensure the endpoint's production code is not someone thrown off by `em1`-s state)
* Let the endpoint-under-test do an `em2.flush`
* Back in the test, run `em1.refresh` so that it can see whatever mutations the endpoint made, and then do assertions like "ah yeah, the author name changed from fred to bob".

If the idea of the tests was to have the `await em.find(Author, ...)` have more of a clean slate, then just creating a 2nd/net-new em is probably the best way to do that.

Reply:
yeah i the intent was a new clean slate. thanks for the heads up